tools/Documentation/api-documentation/miscellaneous-other-api.md

```diff
@@ -283,10 +283,18 @@ You can either use `login`, `metadata`, `script`, or `all` to get all previous t
       {
         "desc": "Save archive title",
         "type": "bool"
+      },
```

Comment:
This wasn't necessary since this is just example output for the API call that returns plugin info, but might as well keep it since it's done. 😁